### PR TITLE
OD-375 [Fix] User is unable to save filter configurations

### PR DESCRIPTION
**Pull request by AndrRyaz**
@sofiiakvasnevska 

## Issue
OD-375 https://weboo.atlassian.net/browse/OD-375

## Description
User is not able to save the filter configurations because there is a wrong if condition for checking the entry value from input
## Screenshots/screencasts
http://somup.com/crnDX3DYot
## Backward compatibility
This change is fully backward compatible.
## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: Medium
Affected: Studio users configuring filter conditions
Verify: Filter options can be saved and loaded to work correctly

Squash and merge: Allow filters be saved with "none" values

File: js/interface-lists.js [hunk 1]
```diff
@@ -506,7 +506,7 @@ function attahObservers() {
               return;
             }
 
-            if (!validate(field.value)) {
+            if (!field.value || !field.value.trim()) {
```
Comment (squallstar) [inline js/interface-lists.js:509]: what if `field.value` isn't a string?

Comment (AndrRyaz) [reply js/interface-lists.js:509]: @squallstar hi, this variable can only be a string with empty value or not empty

**Comment by sofiiakvasnevska** [comment]
deployed to Production on 18/02/21

**Pull request merged**